modules/dynamic-operator/api/v1alpha1/kuberlogicservice_webhook.go

```diff
@@ -60,7 +60,7 @@ func (r *KuberLogicService) Default() {
 		r.Spec.Version = resp.Version
 	}
 	if r.Spec.Domain == "" {
-		r.Spec.Domain = resp.Host
+		r.Spec.Domain = fmt.Sprintf("%s.%s", r.Name, resp.Host)
```

Comment:
Is it related to this issue?

Reply:
@AvoidMe this is incorrect way to fix the bug.

Add this to the validation webhook instead:
```
if domain == "" { return nil }
```

Reply:
Fixed, it's not related to this issue

Reply:
Already on master